modules/dxp/apps/commerce-avalara/commerce-avalara-connector/src/main/java/com/liferay/commerce/avalara/connector/internal/CommerceAvalaraConnectorImpl.java

```diff
@@ -54,6 +84,21 @@ public void verifyConnection(
 		}
 	}
 
+	private AvaTaxClient _getAvaTaxClient() throws PortalException {
+		CommerceAvalaraConnectorConfiguration
```

Comment:
Have you tested with multiple companies to see if everything works as expected?

Reply:
Yes, tested with 3